plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedView.java

```diff
@@ -126,4 +239,99 @@ AS SELECT sum(value) AS s FROM iceberg2.tpch.common_base_table
         assertUpdate(defaultIceberg, "DROP TABLE common_base_table");
         assertUpdate("DROP MATERIALIZED VIEW mv_on_iceberg2");
     }
+
+    private int validateResultContains2AndReturnSecondValue(List<MaterializedRow> materializedRows)
```

Comment:
nit: static

that's a very specialized method. i wish it didn't exist.
you can make it simpler by using ORDER BY in the query
and then inline